src/components/ContentHighlights/ContentHighlightsCardItemsContainer.jsx

```diff
@@ -22,21 +28,39 @@ const ContentHighlightsCardItemsContainer = ({ enterpriseSlug, isLoading, highli
       </Alert>
     );
   }
+  const trackEvent = (metaData = {}) => {
+    const trackInfo = {
+      content_metadata: {
+        aggregation_key: `${metaData?.aggregationKey}`,
+      },
+    };
+    sendEnterpriseTrackEvent(
+      enterpriseId,
+      `${EVENT_NAMES.CONTENT_HIGHLIGHTS.HIGHLIGHT_DASHBOARD_SET_ABOUT_PAGE}.clicked`,
```

Comment:
left a couple instances in since it specified specific buttons within the stepper step.

![Screen Shot 2023-01-11 at 12 30 30 PM](https://user-images.githubusercontent.com/82611798/211876699-43bea797-d16b-4d78-9d84-797707c48c72.png)
![Screen Shot 2023-01-11 at 12 30 40 PM](https://user-images.githubusercontent.com/82611798/211876744-40286249-47f6-4af1-bde5-bd42f63dc42f.png)